test/e2e/node_feature_discovery.go

```diff
@@ -626,7 +626,7 @@ var _ = framework.KubeDescribe("[NFD] Node Feature Discovery", func() {
 				targetNodeName := ""
 				for _, node := range nodeList.Items {
 					for k := range node.Labels {
-						if k == "node-role.kubernetes.io/worker" {
+						if k != "node-role.kubernetes.io/master" {
```

Comment:
There are many labels which are not `node-role.kubernetes.io/master` 😉 

Suggest to do smth like this
```
for _, node := range nodeList.Items {
	if _, ok := node.Labels["node-role.kubernetes.io/master"]; !ok {
		targetNodeName = node.Name
		break
	}
}
```

Reply:
LOL, sorry... I should wait for finishing the 1st coffee in the morning before doing such "trivial changes" :D

Reply:
fixed, thx